.github/workflows/bdd.yml

```diff
@@ -0,0 +1,61 @@
+name: BDD Tests
+on:
+  push:
```

Comment:
you should specify master branch here. There is no need to run integration tests on push to every branch. Only master branch and pull request is sufficient.